### PR TITLE
move all http server into web package

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
Just simply move all the http service into one package

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3920

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by zsluedem** [comment]
bors try

**Review by kayvank — COMMENTED** [review]
@zsluedem, is there a Jira ticket outlining why we need this work?

**Comment by zsluedem** [comment]
@kayvank Updated with the jira ticket.

File: node/src/main/scala/coop/rchain/node/web/BlockInfo.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//package coop.rchain.node.web
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/web/BlockInfo.scala]: Do we need this file for something?

File: node/src/main/scala/coop/rchain/node/web/EventsInfo.scala [hunk 2]
```diff
@@ -10,9 +9,8 @@ import io.circe.Json
 import org.http4s.HttpRoutes
 import org.http4s.server.websocket.WebSocketBuilder
 import org.http4s.websocket.WebSocketFrame
-import org.http4s.websocket.WebSocketFrame._
-
-import scala.{Stream => _}
+import org.http4s.websocket.WebSocketFrame.Text
+import cats.implicits._
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/web/EventsInfo.scala]: Please keep imports formatted.

File: node/src/main/scala/coop/rchain/node/web/ReportingRoutes.scala [hunk 3]
```diff
@@ -1,12 +1,12 @@
-package coop.rchain.node
-
+package coop.rchain.node.web
+import io.circe.syntax._
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/web/ReportingRoutes.scala]: Please keep imports formatted.

File: node/src/main/scala/coop/rchain/node/web/VersionInfo.scala [hunk 4]
```diff
@@ -1,9 +1,9 @@
-package coop.rchain.node
+package coop.rchain.node.web
 
 import cats.effect.Sync
-import cats.implicits._
-
+import coop.rchain.node.BuildInfo
 import org.http4s.HttpRoutes
+import cats.syntax.applicative._
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/web/VersionInfo.scala]: Please keep imports formatted.

File: node/src/main/scala/coop/rchain/node/web/package.scala [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+import coop.rchain.node.NodeRuntime.TaskEnv
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.server.middleware.CORS
+import coop.rchain.node.diagnostics.NewPrometheusReporter
+import coop.rchain.node.effects.EventConsumer
+import org.http4s.implicits._
+import org.http4s.server.Router
+import coop.rchain.metrics.Metrics
+import monix.execution.Scheduler
+import monix.eval.Task
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/web/package.scala]: Please keep imports formatted.

**Review by tgrospic — CHANGES_REQUESTED** [review]
Nice work Will, only few minor things.

File: node/src/main/scala/coop/rchain/node/web/BlockInfo.scala (hunk 1 above)
Comment (zsluedem) [reply node/src/main/scala/coop/rchain/node/web/BlockInfo.scala]: Oh. Sorry . My bad. It should not be included in the pr.

**Review by tgrospic — APPROVED** [review]
LGTM

File: node/src/main/scala/coop/rchain/node/Main.scala [hunk 6]
```diff
@@ -14,6 +14,7 @@ import coop.rchain.metrics.Metrics
 import coop.rchain.node.configuration._
 import coop.rchain.node.diagnostics.Trace
```
Comment (kayvank) [inline node/src/main/scala/coop/rchain/node/Main.scala]: this import is not needed. this pull request does not necessarily need to fix this as its out of scope for this work.

File: node/src/main/scala/coop/rchain/node/NodeRuntime.scala [hunk 7]
```diff
@@ -43,6 +43,7 @@ import coop.rchain.node.diagnostics._
 import coop.rchain.node.diagnostics.Trace.TraceId
 import coop.rchain.node.effects.{EventConsumer, RchainEvents}
 import coop.rchain.node.model.repl.ReplGrpcMonix
+import coop.rchain.node.web._
 import coop.rchain.p2p.effects._
 import coop.rchain.rholang.interpreter.Runtime
 import coop.rchain.rspace.Context
```
Comment (kayvank) [inline node/src/main/scala/coop/rchain/node/NodeRuntime.scala:49]: Please consider removing all org.http4s imports, lines 57 - 61

**Review by kayvank — CHANGES_REQUESTED** [review]

**Comment by zsluedem** [comment]
bors try

**Comment by zsluedem** [comment]
bors try

**Review by kayvank — APPROVED** [review]

**Pull request merged**